docs/tls.md

```diff
@@ -0,0 +1,74 @@
+## Chronograf TLS
+
+Chronograf supports TLS to securely communicate between the browser and server via
+HTTPS.
+
+We recommend using HTTPS with Chronograf.  If you are not using a TLS termination proxy,
+you can run Chronograf's server with TLS connections.
+### TL;DR
+
+```sh
+chronograf --cert=my.crt --key=my.key
+```
+
+### Running Chronograf with TLS
+
+Chronograf server has command line and environment variable options to specify
+the certificate and key files.  The server reads and parses a public/private key
+pair from these files. The files must contain PEM encoded data.
+
+In Chronograf all command line options also have a corresponding environment
+variable. 
+
+To specify the the certificate file either use the `--cert` CLI option or `TLS_CERTIFICATE`
+environment variable.
+
+To specify the key file either use the `--key` CLI option or `TLS_PRIVATE_KEY`
+environment variable.
+
```

Comment:
Maybe unnecessary but, to have all of this kind of information in one place, I might include the text below in this section, and remove `If the cert and the key are in the same file, you don't have to specify the TLS_PRIVATE_KEY option` from the Testing section.

To specify the certificate and key if they're in the same file either use the `--cert` CLI option or `TLS_CERTIFICATE` environment variable.